### PR TITLE
docs: Add lightweight dev testing doc for FeG

### LINKED ISSUE #8179: Add doc in docusaurus for `feg-precommit	`
## Exit Criteria

- [x] Doc coverage for test exists in docusaurus
- [x] Update parent issue (https://github.com/magma/magma/issues/8172) and add a link to the new doc in table

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

## Summary
Add a dev testing section for technical references / feg

## Test Plan
Screen Shot 2021-07-20 at 11 41 12 AM

Screen Shot 2021-07-20 at 11 41 20 AM

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by uri200** [comment]
Nice! 
I have some changes pending on that matter too.
Will try to add them this week.

**Review by uri200 — APPROVED** [review]

**Comment by themarwhal** [comment]
> Nice!
> I have some changes pending on that matter too.
> Will try to add them this week.

Sounds great thanks! I just wanted to provide a baseline change to add some coverage there, but would love it if you could enhance it with more info! :)

**Comment by uri200** [comment]
> > Nice!
> > I have some changes pending on that matter too.
> > Will try to add them this week.
> 
> Sounds great thanks! I just wanted to provide a baseline change to add some coverage there, but would love it if you could enhance it with more info! :)

yep, best way to kick start it

File: docs/readmes/feg/dev_testing.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+id: dev_testing
+title: Testing Tips
+hide_title: true
+---
+
+# Testing Tips
```
Comment (hcgatewood) [inline docs/readmes/feg/dev_testing.md]: I changed the Orc8r to "Test Orc8r", which I think is a clearer title. Wanna change this title as well, to "Test FeG"? https://magma.github.io/magma/docs/next/orc8r/dev_testing

File: docs/readmes/feg/dev_testing.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+This guide covers tips for quickly validating Federation Gateway changes.
+
+
+## Run tests on the host
+
+The normal way to run FeG unit tests is `./$MAGMA_ROOT/feg/gateway/docker/build.py --tests`. This
```
Comment (hcgatewood) [inline docs/readmes/feg/dev_testing.md]: Using `./$MAGMA_ROOT` seems wrong -- remove the `./` ?

**Review by hcgatewood — APPROVED** [review]
Two small notes. Thanks Marie!

**Pull request merged**